### ISSUE TITLE
Multiple agencies in the app don't have nicely corresponding names

### ISSUE THREAD

**Issue by nighelles** [remulasce/metroapp#288]
I.E. "Downtown Berkeley" is the bart stop, while "Berkeley BART..." is the AC Transit stop at the same stop.

Maybe do something to fix that?

**Comment by remulasce** [comment]
This should include the input drop-down as well, so you can see which set of agencies is serving which stops. So if 2 agencies serve the same corner, they probably have extremely similar or exact names, and you want to add the one that you're actually riding on.

Note that Android version currently will consolidate identically named stops from multiple agencies, but it's part of the platform-specific layer.

**Comment by nighelles** [comment]
The issue is when multiple agencies have separate names. But we can
probably deal with that by doing some database editing before runtime.

Also, I strongly agree that there should be a way to tell what agency it is
for a MultiArrivalTrip or something in the search bar

**Comment by remulasce** [comment]
Yeah. I'm not sure how we'd merge stops with separate names though, without doing a ton of manual work. Maybe some proximity-based system. But I can see that going wrong really quickly.

We'll be fighting for space to show the agency everywhere. Can we make like an icon for each agency, so it doesn't take so much room? Too bad none of these agencies allow you to use their branding.

Otherwise, look for ways to fit the greyed-out mino text everywhere, like how the distance-to text is shown.

**Comment by remulasce** [comment]
Your color-coding idea might work, so we would only need to find one place to put the full agency name, and color code the rest of the uses.

**Comment by remulasce** [comment]
I've got the proper-names for agencies displayed in the region settings box

Just as a show of what names are available.
